src/main/java/com/example/crm_gym/controllers/LoginController.java

```diff
@@ -0,0 +1,97 @@
+package com.example.crm_gym.controllers;
+
+import com.example.crm_gym.dao.UserDAO;
+import com.example.crm_gym.models.User;
+import io.swagger.annotations.Api;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@RestController
+@Slf4j
+@RequestMapping("/user")
+@Api(produces = "application/json", value = "Operations for login and updating user's password in the application")
+public class LoginController {
+
+    @Autowired
+    private UserDAO userDao;
+
+    @GetMapping("/login")
+    public ResponseEntity<Map<String, String>> loginUser(
+            @RequestParam("username") String username,
+            @RequestParam("password") String password) {
+        try {
+            Optional<User> userOptional = userDao.findByUsername(username);
```

Comment:
you cand throw from Service EntityNotFoundException and @ControllerAdvice can catch this exception and you won't handle errors here